### PR TITLE
Add explicit reference to new roles/entities...

### PR DESCRIPTION
**Pull request by jfinkhaeuser**
... based on the DD discussion on third-party signers.

The point isn't to model those entities in the core protocol, but to allow for extensions to roles, and to model processes as requiring specific roles.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
Note that the requirements document is not part of the charter, and is not binding to the group (if formed) without some consensus position recording it on the list.

**Comment by rpp-ietf** [comment]
Orie,

I believe the requirements is extremely important and should be part of the charter.  What has been done for other like working groups?   How do we ensure that it is a committed work item for the working group?

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Orie Steele ***@***.***>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.***>
Date: Thursday, December 5, 2024 at 9:18 AM
To: SIDN/ietf-wg-rpp-charter ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] Add explicit reference to new roles/entities... (PR #51)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Note that the requirements document is not part of the charter, and is not binding to the group (if formed) without some consensus position recording it on the list.

**Comment by OR13** [comment]
I answered this off list, but I don't see the reply here.

TLDR; Wikis / other supporting document locations are a fine for the wg to use.

Per:

https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/

I believe it is best to keep the charter focused on the scope and deliverables and leave the specific milestones / our normal process to manage the level of requirements gathering that is needed for each proposed standard or informational draft.

Jim, responded to me saying his comment had been addressed.

I suggest merging this change, even though this document will not appear in the data tracker.

**Comment by jfinkhaeuser** [comment]
The deliverables section mentions a core architecture, and agreement on requirements.

If there is a concern about requirements, we can always do what other WGs do, and make one of the deliverables a more refined requirements doc. Agreement on requirements doesn't strictly speaking imply this deliverable, but it may be useful for the purpose of discussing/reaching consensus.

At any rate, I know why I added it to the requirements doc, not to the charter :)
